### PR TITLE
add user story for multiple manifests

### PR DESCRIPTION
**Pull request by jamesgorrie**
In the light of having a bug when having released the new manifest links, we now have a user story for it, and some e2e tests.

I would like to follow this pattern for when we're updating the viewer as the view has got complex enough now to have to need tests to make sure we capture what is expected.

### CODE REVIEW

File: catalogue/webapp/components/MultipleManifestList/MultipleManifestList.tsx [hunk 1]
```diff
@@ -74,26 +77,31 @@ const MultipleManifestList: FunctionComponent<Props> = ({
...
         isDark
-        aria-controls="hiddenContent"
+        aria-controls={hiddenContentId}
         aria-expanded={showHidden}
         onClick={() => {
           setShowHidden(!showHidden);
         }}
       >
-        <span className={`btn__text`}>{buttonText}</span>
+        <span data-testid="current-manifest" className={`btn__text`}>
```
Comment (gestchild) [inline catalogue/webapp/components/MultipleManifestList/MultipleManifestList.tsx:99]: don't we use data-test-id elsewhere?

File: catalogue/webapp/pages/item.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ import {
 } from '@weco/common/views/pages/_app';
 import * as ItemProps from '@weco/common/views/components/ItemLink/ItemLink';
 import WorkLink from '@weco/common/views/components/WorkLink/WorkLink';
+import { useRouter } from 'next/router';
```
Comment (gestchild) [inline catalogue/webapp/pages/item.tsx:44]: can't see this being used

File: catalogue/webapp/components/MultipleManifestList/MultipleManifestList.tsx (hunk 1 above)
Comment (jamesgorrie) [reply catalogue/webapp/components/MultipleManifestList/MultipleManifestList.tsx]: Yip - changed.

File: catalogue/webapp/pages/item.tsx (hunk 2 above)
Comment (jamesgorrie) [reply catalogue/webapp/pages/item.tsx]: Nice catch, removed.

File: common/views/components/ItemLink/ItemLink.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import NextLink, { LinkProps } from 'next/link';
+import { useRouter } from 'next/router';
```
Comment (gestchild) [inline common/views/components/ItemLink/ItemLink.tsx:2]: needed?

Comment (jamesgorrie) [reply common/views/components/ItemLink/ItemLink.tsx]: Apologies, I was trying something to stop prop drilling, and had some hangover.

File: playwright/user-stories/works-search.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-## A user can complete a works search
+## A researcher can view items on a work in our catalogue
```
Comment (jamesgorrie) [inline playwright/user-stories/works-search.md:1]:
```suggestion
## A researcher can complete a works search
```

**Review by gestchild — APPROVED** [review]

**Pull request merged**